bundles/org.openhab.voice.watsonstt/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0
+		https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="voice:watsonstt">
+		<parameter-group name="authentication">
+			<label>Authentication</label>
+			<description>Information for connection to your Watson Speech-to-Text instance.</description>
+		</parameter-group>
+		<parameter-group name="stt">
+			<label>STT Configuration</label>
+			<description>Parameters for Watson Speech-to-Text API.</description>
+		</parameter-group>
+		<parameter name="apiKey" type="text" required="true" groupName="authentication">
+			<label>Api Key</label>
+			<description>Api key for Speech-to-Text instance created on IBM Cloud.</description>
+		</parameter>
+		<parameter name="instanceUrl" type="text" required="true" groupName="authentication">
+			<label>Instance Url</label>
+			<description>Url for Speech-to-Text instance created on IBM Cloud.</description>
+		</parameter>
+		<parameter name="backgroundAudioSuppression" type="decimal" min="0" max="1" step="0.1" groupName="stt">
+			<label>Background Audio Suppression</label>
+			<description>Use the parameter to suppress side conversations or background noise.</description>
+			<default>0</default>
+		</parameter>
+		<parameter name="speechDetectorSensitivity" type="decimal" min="0" max="1" step="0.1" groupName="stt">
+			<label>Speech Detector Sensitivity</label>
+			<description>Use the parameter to suppress word insertions from music, coughing, and other non-speech events.</description>
+			<default>0.5</default>
+		</parameter>
+		<parameter name="inactivityTimeout" type="integer" unit="s" groupName="stt">
+			<label>Inactivity Timeout</label>
+			<description>The time in seconds after which, if only silence (no speech) is detected in the audio, the connection is
+				closed.</description>
+			<unitLabel>s</unitLabel>
```

Comment:
To be suppressed.

Reply:
I don't understand the comment. Can you add some explanation please?

Reply:
The unit is already defined as an attribute.
What means this tag for ?